code/modules/guardian/guardianbuilder.dm

```diff
@@ -105,10 +105,10 @@
 		return
 	switch(action)
 		if("name")
-			guardian_name = params["name"]
+			guardian_name = sanitize(params["name"])
```

Comment:
This should use reject_bad_name, as that will reject names that aren't allowed, including applying name filtering